meta-iot2000-bsp/recipes-kernel/linux/files/iot2000.cfg

```diff
@@ -34,3 +34,6 @@ CONFIG_IE6XX_WDT=y
 
 CONFIG_IP_NF_NAT=y
 
+CONFIG_R8712U=y
+CONFIG_RTL8XXXU=y
+CONFIG_RTL8192CU=y
```

Comment:
Not really a modular configuration, like the subject suggests. Rather, this builds the driver into the kernel. What is actually desired?